app/components/filmstrip.cjsx

```diff
@@ -0,0 +1,89 @@
+React = require 'react'
+{Link, State, Navigation} = require 'react-router'
+{DISCIPLINES} = require './disciplines'
+
+module.exports = React.createClass
+
+  displayName: 'Filmstrip'
+
+  propTypes:
+    increment: React.PropTypes.number.isRequired
+    onChange: React.PropTypes.func.isRequired
+
+  getDefaultProps: ->
+    filterCards: DISCIPLINES
+
+  getInitialState: ->
+    scrollPos: 0
+
+  scrollLeft: ->
+    @adjustPos(-@props.increment)
+
+  scrollRight: ->
+    @adjustPos(@props.increment)
+
+  mangleFilterName: (filterName) ->
+    filterName.replace(/\s+/g,'-')
+
+  selectFilter: (filterName) ->
+    @props.onChange filterName
+
+  calculateClasses: (filterName)->
+    filterName = @mangleFilterName(filterName)
+
+    list = ['filmstrip--disciplines__discipline-card']
+    list.push "filmstrip--disciplines__discipline-card--discipline-#{filterName}"
+
+    if(@props.selectedFilter == filterName)
+      list.push 'filmstrip--disciplines__discipline-card--active'
```

Comment:
This is a nice-sized standalone piece of the site, so I asked Amy to try implementing [BEM](http://getbem.com/introduction/). Our CSS is out of control, and BEM was designed to get teams on the same page syntax-wise to avoid conflicts. If her experience was good, we should start using this across the site.